homeassistant/components/solarlog/coordinator.py

```diff
@@ -45,8 +45,14 @@ def __init__(self, hass: HomeAssistant, entry: SolarlogConfigEntry) -> None:
 
         extended_data = entry.data["extended_data"]
 
+        enabled_devices: dict[int, bool] = {}
+        if extended_data:
+            if entry.data.get("devices", {}) != {}:
+                for key, value in entry.data["devices"].items():
+                    enabled_devices |= {int(key): value}
+
```

Comment:
What does this do?

Reply:
Initially the idea was to only fetch the data for enabled devices. Now with all devices created anyway, this indees becomes obsolete, as all data will be retreived. Will have to think how to deal with this with the library...